cnc_ctrl_v1/CNC_Functions.h

```diff
@@ -22,6 +22,11 @@ libraries*/
 
 #define VERSIONNUMBER 0.89
 
+#define verboseDebug 2    // set to 0 for no debug messages, 1 for single-line messages, 2 to also output ring buffer contents
+
+#include <Servo.h>
+Servo myservo;  // create servo object to control a servo 
```

Comment:
Why is there a servo involved? 😀

Reply:
Oh! I get it! It's just and easy way to get a PWM output! 

The servo library might be affected by some of the timers we're using for computing the PID loops, just something to keep an eye on

Reply:
Actually, I'm experimenting with using a servo to physically turn a switch on and off to control power to the router.  I was concerned about possible impact to timers which is why I wanted to test this first.  For minimal impact, I attach() the servo, use it, then detach() it, so hopefully there is no impact to timers.  I didn't see an issue when I ran this on my Maslow two days ago.

I also included the option to use a digital I/O pin to control a relay.  I know you're reluctant to have the Maslow control power to the router, so I wasn't planning on rolling this out just yet.  But if we make the defaults "controlPin - AUX1", "useServo = false", and "delayAfterChange = 0", there should be no time penalty, and no impact as long as AUX1 isn't connected to anything.